contrib/twij

```diff
@@ -70,7 +63,64 @@
 #   -X: Abort (with exit code 4) if a delay would be greater than the
 #       maximum specified by -T.
 #
-# Copyright © 2015 Decklin Foster <decklin@red-bean.com>; distributed
+# Examples:
+#
+# This first set is overly simplified, but will give you a feel for what
+# you can do.
+#
+#     # Read the latest tweets in your home timeline, formatted
+#     twij /statuses/home_timeline | jq -r '.[] | "<\(.user.screen_name)> \(.text)"'
+#
+#     # List the names of all accounts @twitterapi is following (all in one request)
+#     twij /friends/list screen_name=twitterapi count=200 | jq -r '.users[].screen_name'
+#
+#     # Do the same, but inefficiently, with verbose logging of the multiple requests
+#     twij -v /friends/list screen_name=twitterapi | jq -r '.users[].screen_name'
+#
+# In practice, you want to be able to handle and recover from errors
+# (see the next section) -- here's the same inefficient version, but but
+# we'll know when our rate limit is used up:
+#
+#     if twij -v -x /friends/list screen_name=twitterapi > temp.jsonl; then
+#         jq -r '.users[].screen_name' temp.jsonl
+#     fi
+#
+# Or, to only give up if the next rate limit window is more than 5 minutes away:
+#
+#     if twij -v -T 300 -X /friends/list screen_name=twitterapi > temp.jsonl; then
+#         jq -r '.users[].screen_name' temp.jsonl
+#     fi
+#
+# Caveat:
+#
+# The simple example we start out with pipes twij into jq, which looks
+# easy, and is probably good enough for one-off tasks, but if you are
+# doing anything that might hit a rate limit, it's a dangerous idea.
```

Comment:
Can I suggest changing `it's a dangerous idea` to something more like `it could lead to accidentally and quietly running a much narrower search than intended`?